build.gradle

```diff
@@ -302,7 +300,7 @@ project(':iceberg-presto-runtime') {
         shadow "org.apache.avro:avro:$avroVersion"
         shadow ("org.apache.hive:hive-standalone-metastore:$hiveVersion") {
             exclude group: 'org.apache.hadoop', module: 'hadoop-common'
-            exclude group: 'org.apache.orc', module: 'orc-core'
+//            exclude group: 'org.apache.orc', module: 'orc-core'
```

Comment:
Hi, @rdblue . It's great to see Spark 2.4 support.
So, are you going to file a GitHub issue to bring ORC back?

Reply:
@dongjoon-hyun We'd really appreciate some support getting ORC back in.  The current implementation is incomplete and rather than provide inconsistent behavior, it's probably best to remove until the implementation is complete.

Reply:
I agree with Dan. We would love to see ORC support, but I don't think any of the current committers has time to invest in it right now. I'm happy to work with someone to fix the ORC support, though.

I also need to move the issues that track what wasn't implemented from the old repository and add an ORC tag.

Reply:
Got it. Thank you for the replies, @danielcweeks and @rdblue .